librz/io/p/io_srec.c

```diff
@@ -0,0 +1,802 @@
+// Copyright: Manolache183 <manolache.alexandru8@gmail.com>
+
+/*
+*** SREC format description : every line follows this pattern
+	S / Type / Bytecount / Address / Data / Checksum
+
+	There are a bunch of types, from which one (S4) is reserved
+
+//sauce : https://en.wikipedia.org/wiki/SREC_(file_format)
```

Comment:
Please 1) put space between comment and word 2) use less informal word like `source` instead.

Reply:
Oh, I am sorry, I saw "sauce" in other files so I just wrote it here without thinking much about it.